README.md

```diff
@@ -1,14 +1,18 @@
 # Bro & VAST
 
-This package enables Bro to communicate with [VAST][vast], a scalable plaform
-for network forensics.
+This package enables Bro to leverage the [`bro-to-vast`][bro-vast] tool for
+querying [VAST][vast] from Bro scripts.
```

Comment:
I actually find the first sentence now more confusing. From a user perspective, I want to speak with VAST, not the helper utility `bro-to-vast`. Therefore I'd prefer to keep it as is.